<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>671</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>CGV ATS</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image_button/resource.qrc">
    <normaloff>:/icon/cgv_icon.ico</normaloff>:/icon/cgv_icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="dropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame{
	
	
	background-color: rgb(48, 53, 103);
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>61</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 170, 255);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;CGV ATS&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="quitButton">
       <property name="geometry">
        <rect>
         <x>590</x>
         <y>10</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 53, 103);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="image_button/resource.qrc">
         <normaloff>:/icon/icons8-close-64.png</normaloff>:/icon/icons8-close-64.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>421</width>
         <height>521</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>30</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">ID</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_pw">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI Semibold</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">PASSWORD</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_name">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>130</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">영화제목</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_mode">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>180</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">영화모드(전체, 무대인사 등)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_city">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>230</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">도시</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_hall">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>280</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">상영관(강남, 천호 등)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_date">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>330</y>
          <width>110</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">날짜(20XX0X0X)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_person">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>380</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">인원(명)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_row">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>430</y>
          <width>240</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">열(알파벳 대문자)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_seat_1">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>480</y>
          <width>110</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">좌석번호</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_clock">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>330</y>
          <width>110</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>8</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">시간(2145)</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_seat_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>480</y>
          <width>110</width>
          <height>30</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>맑은 고딕</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
	border: 1px solid rgb(88, 98, 188);
	border-radius: 10px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(52, 57, 111);
}
QLineEdit:hover {
	border: 2px solid rgb(62, 68, 132);
}
QLineEdit:focus {
	border: 2px solid rgb(72, 144, 216);
	background-color: rgb(64, 72, 139);
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string notr="true">좌석번호</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="okButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>560</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: rgb(255, 170, 255);
	font: 63 15pt &quot;Segoe UI Semibold&quot;;
	color: rgb(255, 255, 255);
	border-radius: 10px;
}
QPushButton:hover {
	border: 3px solid rgb(204, 136, 204);
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="image_button/resource.qrc"/>
 </resources>
 <connections/>
</ui>
